localstack/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "1.4.1.dev"
+__version__ = "1.4.1.dev1"
```

Comment:
this should remain `1.4.1.dev` i think. it signifies that this is not the concrete version number